packages/untp-test-suite/README.md

```diff
@@ -59,3 +59,67 @@ yarn create:config
 ```bash
 $ yarn test
 ```
+
+---
+
+## UNTP Test Suite CLI Tool
+
+This CLI tool is designed to facilitate the management and execution of the UNTP test suite.
+
+### Installation
+
+Navigate to the `untp-test-suite` package folder. Type the following command into the terminal to install the UNTP Test Suite CLI Tool:
+
+  ```bash
+  npm install -g .
+  ```
+
+Alternatively, you can run the UNTP test suite by typing the following command into the terminal console:
+
+  ```bash
+  npm run untp
+  ```
+
+### **Create Credentials File**
+
+  This command generates a `credentials.json` file in the current working directory.
+
+  ```bash
+  untp config
+  ```
+
+  or
+
+  ```bash
+  npm run untp config
+  ```
+
+  The `credentials.json` file contains configurations for running the test suite, including types and versions of events and their data paths.
+
+---
+
+### Running UNTP Test Suite
+
+  This command executes the UNTP test suite using the default `credentials.json` file in the current working directory.
+
+  ```bash
+  untp test
+  ```
+
+  or
+
+  ```bash
+  npm run untp test
```

Comment:
It's ok when we use npm, but now in our project, we are using yarn. Should we change from npm to yarn? Because when I check the version of the package with npm, I have to use `npm run untp -- --version` instead of with yarn, I just use `yarn run untp --version`

Reply:
It depends on the package manager you are using on your local machine to run the UNTP test suite. Most commonly, people use the npm package manager, I think. So I cannot document all the package manager libraries.
I believe that when developers use this UNTP test suite, they already know how to use the package manager on their local machine.

By the way, we have another way to run it that using the CLI tool. You can try it if you prefer.